auto sync 2022-03-03

nunnatsa opened the pull request:
auto sync upstream 2022-03-03

JoelSpeed commented:
There's an issue updating this at the moment it seems. An AWS dependency is broken in MAO breaking this build. Will close this and create a new PR when that's resolved.

nirarg committed:
Change OWNERS file

nunnatsa committed:
Automatic sync with upstream

This PR adds a new github action to daily sync with upstream. It checks if there are new changes in upstream, and if there are, it creates a PR to sync.

The new PR is created by hard reseting to the upstream main branch, then restoring the downstream modified files.

Signed-off-by: Nahshon Unna-Tsameret <nunnatsa@redhat.com>
